pkg/api/admin/routes.go

```diff
@@ -0,0 +1,42 @@
+package admin
+
+import (
+	"github.com/gofiber/fiber/v2"
+
+	"github.com/G-Research/fasttrackml/pkg/api/admin/controller"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+)
+
+// List of route prefixes.
+const (
+	NamespacesRoutePrefix = "/ns"
+)
+
+// List of `/namespaces/*` routes.
+const (
+	NamespacesListRoute = "/list"
+)
+
+// Router represents `mlflow` router.
+type Router struct {
+	controller *controller.Controller
+}
+
+// NewRouter creates new instance of `mlflow` router.
+func NewRouter(controller *controller.Controller) *Router {
+	return &Router{
+		controller: controller,
+	}
+}
+
+// Init makes initialization of all `mlflow` routes.
+func (r Router) Init(server fiber.Router) {
```

Comment:
Can we at least get an integration test here?

Reply:
yes we can. when we create actual logic for that we can add `integration` tests as well.

Reply:
this is covered by integration tests which use the routes, right?